src/UWP.Images/UWP.Images.projitems

```diff
@@ -9,8 +9,6 @@
     <Import_RootNamespace>UWP.Images</Import_RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <Content Include="$(MSBuildThisFileDirectory)android.png" />
```

Comment:
I believe there are cases when these could be invoked still

Reply:
Shouldn't it have given an error during build then?

Reply:
It would be a runtime error if the image was trying to be displayed and the file does not exist.

Reply:
Ok, I can add it back. Just the Android one?

Reply:
Android and apple are both used in cases when the URI uses their app's scheme

Reply:
Ok, I'll add those back in.